portal/styles/globals.css

```diff
@@ -49,6 +49,12 @@ a {
   grid-template-columns: 10% 90%;
 }
 
+@media screen and (min-width: 768px) {
+  .logo-card:last-child {
+    transform: translateX(50%);
+  }
+}
+
```

Comment:
@sagar-techjoomla - Can you check if this breaks the styles on any other pages?

Reply:
.logo-card this class only using on landing page and school page and its not breaks the styles on this pages